pandas/core/internals.py

```diff
@@ -840,7 +840,7 @@ def setitem(self, indexer, value, mgr=None):
 
         transf = (lambda x: x.T) if self.ndim == 2 else (lambda x: x)
         values = transf(values)
-        l = len(values)
+        vlen = len(values)
```

Comment:
It looks like this is only used in one place (line 858).  Is there a reason to make it a variable in this case?  Seems like we could just replace the one instance where `vlen` is used with `len(values)`.

Reply:
Sounds good, will change.